src/sigchain/sigchain.iced

```diff
@@ -425,8 +446,19 @@ exports.SigChain = class SigChain
 
     # This constructor checks that the link is internally consistent: its
     # signature is valid and belongs to the key it claims, and the same for any
-    # reverse sigs.
-    await ChainLink.parse {sig_blob, key_state: @_key_state, sig_cache}, esc defer link
+    # reverse sigs. Note that we need to explicitly check the error here, in
+    # case the sig is a known buggy one that we intend to skip.
+    await ChainLink.parse {sig_blob, key_state: @_key_state, sig_cache, log}, defer err, link
+    if err?
```

Comment:
Why is it necessary to short-circuit chain integrity here?  As far as I know, Fred's chain still parses fine w/r/t prev links.